### PR TITLE
Add am et

### PR DESCRIPTION
**Pull request by melkam5**
I tried my best

### CODE REVIEW

File: lang/am-ET.json [hunk 1]
```diff
@@ -1,120 +1,120 @@
...
-  "metrics.referrers": "Referrers",
-  "metrics.screens": "Screens",
-  "metrics.unique-visitors": "Unique visitors",
-  "metrics.views": "Views",
-  "metrics.visitors": "Visitors"
+  "label.accounts": "መለያዎች",
+  "label.add-account": "መለያ ጨምር",
+  "label.add-column": "አምድ ጨምር",
+  "label.add-filter": "ማጣሪያ ጨምር",
+  "label.add-website": "ድር ጣቢያ ጨምር",
```
Comment (SamAsEnd) [inline lang/am-ET.json]: ድር ጣቢያ?

File: lang/am-ET.json [hunk 2]
```diff
@@ -1,120 +1,120 @@
...
+  "label.administrator": "አስተዳዳሪ",
+  "label.all": "ሁሉም",
+  "label.all-time": "ሁሌ",
```
Comment (SamAsEnd) [inline lang/am-ET.json]: Need more context

File: lang/am-ET.json [hunk 3]
```diff
@@ -1,120 +1,120 @@
...
+  "label.all-websites": "ሁሉም ድር ጣቢያዎች",
+  "label.back": "ተመለስ",
+  "label.cancel": "ሰርዝ",
+  "label.change-password": "የይለፍ ቃል ቀይር",
+  "label.confirm-password": "የይለፍ ቃል አረጋግጥ",
+  "label.copy-to-clipboard": "ወደ ቅንጥብ ሰሌዳ ቅዳ",
```
Comment (SamAsEnd) [inline lang/am-ET.json]: ቅንጥብ? Just use ክሊፕቦርድ

File: lang/am-ET.json [hunk 4]
```diff
@@ -1,120 +1,120 @@
...
+  "label.current-password": "የአሁኑ የይለፍ ቃል",
+  "label.custom-range": "ብጁ ክልል",
```
Comment (SamAsEnd) [inline lang/am-ET.json]: 👀

File: lang/am-ET.json [hunk 5]
```diff
@@ -1,120 +1,120 @@
...
+  "metrics.filter.combined": "የተዋሃደ",
+  "metrics.filter.raw": "ጥሬ",
+  "metrics.languages": "ቋንቋዎች",
+  "metrics.operating-systems": "ስርዓተ ክወናዎች",
+  "metrics.page-views": "የገጽ እይታዎች",
+  "metrics.pages": "ገፆች",
+  "metrics.query-parameters": "የጥያቄ መለኪያዎች",
+  "metrics.referrers": "አጣቃሾች",
+  "metrics.screens": "ስክሪኖች",
+  "metrics.unique-visitors": "ልዩ ጎብኝዎች",
```
Comment (SamAsEnd) [inline lang/am-ET.json:117]: ?

File: lang/am-ET.json (hunk 1 above)
Comment (melkam5) [reply lang/am-ET.json]: I was thinking "ድር ጣቢያ" for "website" and "ድህረገፅ" for "webpage"

**Pull request closed without merging**